flask_praetorian/base.py

```diff
@@ -644,6 +654,35 @@ def read_token_from_header(self):
         """
         return self._unpack_header(flask.request.headers)
 
+    def _unpack_cookie(self, cookies):
+        """
+        Unpacks a jwt token from a request cookies
+        """
+        jwt_cookie = cookies.get(self.cookie_name)
+        MissingToken.require_condition(
+            jwt_cookie is not None,
+            "JWT token not found in cookie under '{}'",
+            self.cookie_name,
+        )
+        return jwt_cookie
+
+    def read_token_from_cookie(self):
+        """
+        Unpacks a jwt token from the current flask request
+        """
+        return self._unpack_cookie(flask.request.cookies)
+
+    def read_token(self):
+        exc = None
+        for method in self.cookie_places:
```

Comment:
Since there are only 2 places where a jwt might be, I would rather just explicitly try both instead of looping. I don't anticipate extending to other types of token bearers than headers and cookies.